src/components/PageHeader/_PageHeader.scss

```diff
@@ -1,8 +1,20 @@
 .coa-PageHeader {
-  margin-bottom: $coa-spacing-large;
-  padding-bottom: 1px; //prevent margin-stacking
+  // margin-bottom: $coa-spacing-large;
```

Comment:
same here, does this style get reflected elsewhere, or is it no longer relevant?

Reply:
It's no longer relevant. There are a some stylistic modifiers/differences for the PageHeader component depending on the content-type (service, information, topic) that will have to be accounted for soon.

Reply:
sounds good; maybe we should stick a brief note explaining that?